test/e2e/operator.go

```diff
@@ -0,0 +1,95 @@
+package e2e
+
+import (
+	"os"
+	"testing"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+)
+
+// regularTestCases contains all data in samples, the strategy shipped by the operator are verified
+// during CI.
+var regularTestCases = map[string]*SampleFiles{
+	"buildah": {
+		ClusterBuildStrategy: "samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml",
+		Build:                "samples/build/build_buildah_cr.yaml",
+		BuildRun:             "samples/buildrun/buildrun_buildah_cr.yaml",
+	},
+	"buildah-custom-context-dockerfile": {
+		ClusterBuildStrategy: "samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml",
+		Build:                "test/data/build_buildah_cr_custom_context+dockerfile.yaml",
+		BuildRun:             "test/data/buildrun_buildah_cr_custom_context+dockerfile.yaml",
+	},
+	"buildpacks-v3": {
+		ClusterBuildStrategy: "samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml",
+		Build:                "samples/build/build_buildpacks-v3_cr.yaml",
+		BuildRun:             "samples/buildrun/buildrun_buildpacks-v3_cr.yaml",
+	},
+	"buildpacks-v3-namespaced": {
+		BuildStrategy: "samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_namespaced_cr.yaml",
+		Build:         "samples/build/build_buildpacks-v3_namespaced_cr.yaml",
+		BuildRun:      "samples/buildrun/buildrun_buildpacks-v3_namespaced_cr.yaml",
+	},
+	"kaniko": {
+		ClusterBuildStrategy: "samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml",
+		Build:                "samples/build/build_kaniko_cr.yaml",
+		BuildRun:             "samples/buildrun/buildrun_kaniko_cr.yaml",
+	},
+	"kaniko-custom-context-dockerfile": {
+		ClusterBuildStrategy: "samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml",
+		Build:                "test/data/build_kaniko_cr_custom_context+dockerfile.yaml",
+		BuildRun:             "test/data/buildrun_kaniko_cr_custom_context+dockerfile.yaml",
+	},
+	"s2i": {
```

Comment:
As https://github.com/redhat-developer/build/pull/118 was merged, you need to add the test case `kaniko-advanced-dockerfile`.

```go
	"kaniko-advanced-dockerfile": {
		ClusterBuildStrategy: "samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml",
		Build:                "test/data/build_kaniko_cr_advanced_dockerfile.yaml",
		BuildRun:             "test/data/buildrun_kaniko_cr_advanced_dockerfile.yaml",
	},
```

Reply:
Thanks for the heads-up!

Reply:
Included the test case mentioned, please consider.